apps/question-bank/src/test/java/gov/cdc/nbs/questionbank/pagerules/PageRuleServiceImplTest.java

```diff
@@ -454,4 +458,58 @@ void shouldCreateJsForEnableAndDisableFunction() {
         Assert.assertEquals(jsonString, jsFunctionNameHelper.jsFunction());
     }
 
+    @Test
+    void getAllPageRuleTest() {
+        Pageable pageable= null;
+        Page<WaRuleMetadata> ruleMetadataPage= new PageImpl<>(Collections.singletonList(morbWaRuleMetadata()));
+        Mockito.when(waRuleMetaDataRepository.findAll(pageable)).thenReturn(ruleMetadataPage);
+        Page<ViewRuleResponse> ruleResponse = pageRuleFinderServiceImpl.getAllPageRule(pageable);
+        assertNotNull(ruleResponse);
+        List<ViewRuleResponse> actualResponse = ruleResponse.stream().toList();
+        assertNotNull(actualResponse);
+        assertEquals(1,actualResponse.size());
+        assertEquals(99,actualResponse.get(0).ruleId());
+        assertEquals("Question",actualResponse.get(0).targetType());
+
+        WaRuleMetadata data = morbWaRuleMetadata();
+        data.setSourceValues("Test Source values");
+        data.setTargetQuestionIdentifier("Test target questions identifier");
+        ruleMetadataPage= new PageImpl<>(Collections.singletonList(data));
+        Mockito.when(waRuleMetaDataRepository.findAll(pageable)).thenReturn(ruleMetadataPage);
+        ruleResponse = pageRuleFinderServiceImpl.getAllPageRule(pageable);
+        assertNotNull(ruleResponse);
+        actualResponse = ruleResponse.stream().toList();
+        assertNotNull(actualResponse);
+        assertEquals(1,actualResponse.size());
+        assertEquals(99,actualResponse.get(0).ruleId());
+    }
```

Comment:
Similar to Adam's comment above, would be good to verify that the `ViewRuleResponse` fields match what was in the `WaRuleMetadata` object.